resources/validations/README.md

```diff
@@ -1,72 +1,76 @@
-
-`-f` *\<required>* is the input file to be tested. ex: `-f test/demo/FedRAMP-SSP-OSCAL-Template.xml`
-
-`-s` *\<optional>* schematron directory used to validate the file. Each .sch found within the specified directory will be compliled and generate a separate report. defaults to src relative to the parent of the bin directory where this script is located.  ex: `-o ~/mySchematronDirectory`
-
-`-o` *\<optional>* is the root directory of the report output. ex: `-o ~/dev/report`
-
-`-b` *\<optional>* specifies the base directory of the location of this project (for relative references to target, bin and dependencies like OSCAL definiitions). defaults to `.` ex: `-b /dev/fedramp-automation/resources/validations`
-
-`-v` *\<optional>* if you wish to override the default version (currently 10.2) of `SAXON HE`, that is downloaded and used if $SAXON_CP is not specified. ex:  `-v 10.2.2` *Note,  SAXON_CP is set as an Environment Variable. and `-v` is specified, the script will terminate due to inability to determine priority.*
-
-`-h` *\<optional>* outputs the help/usage for the script.`
-
-example:
-
-`./bin/validate_with_schematron.sh -f test/demo/FedRAMP-SSP-OSCAL-Template.xml -o ~/dev/report -v 10.2.2`
-
-To Run Unit Tests
----
-
-*Prerequesite
-if you haven't done it previously: to add the needed dependencies (declared by .gitmodules), run the following:*
-
-`git submodule update --init --recursive`
-
-```sh
-cd /path/to/fedramp-automation/resources/validations
-#if you have a preferred version of a saxon jar downloaded export SAXON_CP as so
-export SAXON_CP=yourpath/Saxon-HE-X.Y.Z.jar
-#set the test directory relative to project path, you may change if you prefer somehere else
-export TEST_DIR=$(pwd)/report/test
-#execute xpec with the test harness that runs all tests
-lib/xspec/bin/xspec.sh -s -j test/test_all.xspec
-```
-
-Adding tests to the harness
----
-
-To add new tests, add an import to the `test-all.xpec`
-ex: `<x:import href="new_test.xspec" />`
+# FedRAMP Validation Suite
+
+## What is this?
+
+This is the FedRAMP Validation Suite, a framework to take [FedRAMP documentation](https://www.fedramp.gov/templates/) that is properly formatted with [NIST OSCAL schemas](https://pages.nist.gov/OSCAL/) and check the content for correctness. [FedRAMP's adoption of OSCAL](https://www.youtube.com/watch?v=WCPkt56vZ-s) allows you to use this framework to perform logical validations, i.e. business rule checks, on documentation content. Currently, FedRAMP staff manually review the content provided by a cloud service provider for all steps of the [different](https://github.com/GSA/fedramp-gov/blob/master/assets/img/agency-auth.png) [authorization](https://github.com/GSA/fedramp-gov/blob/master/assets/img/ato-auth.png) processes. This framework and the associated business rules will automate as many of these checks as possible. Shared expectations are the goal: system owners, third-party assessors, or any interested party can use them as well.
```

Comment:
I intentionally used the words "formatted with" to avoid confusion between schema validation (like XSD) and what Schematron rules we do are. This might be somewhat obtuse. Feedback welcome!

Reply:
I think this is a good distinction to make. I can see this readme incorporate a link to (or a marked-up version) of [this](https://files.gitter.im/5f933eedd73408ce4ff253cb/f2KX/Adjudication_Approach_DRAFT_02.png). The reason to potentially 'mark it up' would be to indicate where in the process flow this instrumentation lives (think "you are here"). Additionally, when the adjudication extension is adopted, updating the readme to switch any vernacular that is specific to the model will further strengthen the concept.